release-handbook/doc_roles_handbook.md

```diff
@@ -0,0 +1,37 @@
+# Zowe documentation squad roles
+
+This document covers the responsibilities, time commitments, and basic processes and guidance for different roles within the Zowe Docs team. 
+
+## Zowe docs roles
+
+- Zowe Docs Lead
+- Zowe Docs Scrum Master
+- Zowe Docs Lead Shadows/Co-lead
+
+## Docs Lead Responsibilities
+
+The Docs Lead is responsible for coordinating documentation updates for new Zowe releases, manaing the Zowe docs-site repo, and leading the squad to define documentation roadmap and strategy.
+
+Responsibilities include:
+
+Task | Where | Time requirements | Resources, comments
+---| ---| ---| ---
+Monitor Zowe docs-site issues and assign owners | Zowe docs-site GitHub repo | 1-2 hours a week | Looking to automate the process
+Monitor Zowe doc-site Pull Requests and assign reviewers | Zowe docs-site GitHub repo | 1 hour a week | 
+Identify and track new Zowe documentation requests and feedback in other channels like Slack, provide response, and convert into actionable issues | Slack, meetings where invited, emails | Depends, usually 2-3 hours a week | As a squad lead, you'll be the go-to-person whenever people have questions, comments, or ideas about documentation so might get involved in different discussions.
+Offer guidance to doc contributors about the doc site process and answer questions | Slack, GitHub repo | On demand, usually <1 hour a week depending on the complexity of help needed | New developers in other squads are also doc contributors. Introduce new contributors to the doc process and empower them with the knowledge needed to get started.
+Coordinate release activities for docs, communicate the release schedule, monitor all release tasks to be completed on time | Zowe docs-site repo, Slack | 2-3 hours per release | In general, there's a lot of work in the last week of the release cycle as documentation deadline approaches.
+Maintain and optimize documentation processes and update the instructions | Zowe docs-site repo | Depends | Release handbook, maintainer handbook, contributing guidelines, release process, PR review guide, issue triage guide, etc. 
+Identify overall documentation improvement areas and lead the squad to discuss and take action | / | Depends | For example, overall architectural change that requires squad collaboration and discussion, initiatives from other squads that need doc participation. 
+Join Zowe quarterly webinar to present Zowe doc changes | Zowe webinar | On demand | Onboarding squad will contact the lead if there is need for doc squad to join. 
+Participate in Zowe PI planning, present the squad objectives and bring back feedback | Zowe PI planning sessions | 3 days for each PI | The squad lead also needs to schedule breakout sessions and lead the squad members to discuss and finalize the objectives. 
+
+
```

Comment:
I'd also add reviewing traffic metrics/patters for the Zowe Docs website. (And are any special permissions or credentials needed for that.)